blackd.py

```diff
@@ -105,10 +108,25 @@ def make_app() -> web.Application:
         req_bytes = await request.content.read()
         charset = request.charset if request.charset is not None else "utf8"
         req_str = req_bytes.decode(charset)
+        then = datetime.utcnow()
+
         loop = asyncio.get_event_loop()
         formatted_str = await loop.run_in_executor(
             executor, partial(black.format_file_contents, req_str, fast=fast, mode=mode)
         )
+
+        # Only output the diff in the HTTP response
+        only_diff = bool(request.headers.get(DIFF_HEADER, False))
+        if only_diff:
+            now = datetime.utcnow()
+            src_name = f"In\t{then} +0000"
+            dst_name = f"Out\t{now} +0000"
+            loop = asyncio.get_event_loop()
+            formatted_str = await loop.run_in_executor(
```

Comment:
I wonder if we can just run `black.diff` synchronously here instead of delegating to a process pool.

For short files it's going to be probably faster, but if it takes too long it will block the event loop.

Reply:
I think I convinced myself that this is probably fine for now, we can be more sophisticated later if it really is too slow